iroh-relay/src/client/conn.rs

```diff
@@ -325,7 +314,7 @@ impl Sink<SendMessage> for Conn {
     fn start_send(mut self: Pin<&mut Self>, item: SendMessage) -> Result<(), Self::Error> {
         if let SendMessage::SendPacket(_, bytes) = &item {
             if bytes.len() > MAX_PACKET_SIZE {
-                return Err(ConnSendError::Protocol("Packet exceeds MAX_PACKET_SIZE"));
+                return Err(ExceedsMaxPacketSizeSnafu { size: bytes.len() }.build());
```

Comment:
you can use snafu::ensure for these checks

Reply:
done